tests/integration/motions/test_viewset.py

```diff
@@ -113,9 +113,13 @@ def test_with_multiple_comments(self):
         comments = ['comemnt1_sdpoiuffo3%7dwDwW&', 'comment2_iusd&D/TdskDWH&5DWas46WAd078']
         response = self.client.post(
             reverse('motion-list'),
-            {'title': 'title_test_sfdAaufd56HR7sd5FDq7av',
-             'text': 'text_test_fiuhefF86()ew1Ef346AF6W',
-             'comments': comments})
+            {
+                'title': 'title_test_sfdAaufd56HR7sd5FDq7av',
+                'text': 'text_test_fiuhefF86()ew1Ef346AF6W',
+                'comments': comments,
+            },
+            format='json',
```

Comment:
@normanjaeckel No, this doesn't work, but i do not know why. I mean it should be working..